server/src/interfaces/surveyModel.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface SurveyResultRequestModel {
```

Comment:
No need of moving these model classes from under the src/models, I would rather suggest to create the src/db/models to handle the db entity.